Add: Sigma Dao Screenshots (#61)
scale-it/sigmadao-ui: README.md

<img src="./src/assets/logo.png" height="200" title="SigmaDAO" />

# SigmaDAO

> SigmaDAO is a web application that let's a user create a DAO, enable them to Add Proposal, Deposit and Withdraw Tokens and also allow them to Vote.

Reviewer: ```suggestion
SigmaDAO is a web application that lets anyone create a DAO, enable them to Add Proposal, Deposit and Withdraw Tokens and also allow them to Vote.
```

> Wallets supported by SigmaDAO are:

Reviewer: ```suggestion
Wallets supported by the SigmaDAO are:
```

<img src="./src/assets/logo.png" height="200" title="Sigma Dao" />

> # SigmaDao Glimpes:

Reviewer: ```suggestion
## SigmaDAO glimpes:
```
